core/paths/paths.go

```diff
@@ -178,3 +184,49 @@ func MustSucceed(s string, e error) string {
 	}
 	return s
 }
+
+func SetOwnerGroupLog(filePath string, wantedOwner string, wantedGroup string) error {
```

Comment:
This needs a function docstring, and perhaps a rename?

What about `SetOwnership`? wantedOwner is really wantedUser if you are worried about confusion.